lasio/reader.py

```diff
@@ -2,12 +2,10 @@
 import logging
 import os
 import re
-import textwrap
 import traceback
 
 import numpy as np
 
-from . import defaults
```

Comment:
This is used in reader.py:

- https://github.com/kinverarity1/lasio/blob/b001ae1d92ac2afdaa77787b0d3110a82fc60ac3/lasio/reader.py#L464-L465
- https://github.com/kinverarity1/lasio/blob/b001ae1d92ac2afdaa77787b0d3110a82fc60ac3/lasio/reader.py#L600
- https://github.com/kinverarity1/lasio/blob/b001ae1d92ac2afdaa77787b0d3110a82fc60ac3/lasio/reader.py#L639

Reply:
Yes, but it was imported twice. Once at line 10 and then at line 30. I had removed the first import.
Not sure why Python2.7 build is failing, it is not starting the tests. 
But failing for command "python setup.py egg_info:" after taking ~8 mins.

Reply:
Oh whoops - thanks for picking that up, we can definitely remove the duplicate line in that case. Sorry.